contrastive_training/contrastive_main.py

```diff
@@ -49,23 +49,23 @@
 #input_mash_mat = torch.from_numpy(input_mash_mat_np).to(dtype=dtype,device=device)
 #data_size = input_mash_mat.shape[0]
 
-input_mash_mat_subsample_np = np.load("/contrastive_training/input_features.npy")
+input_mash_mat_subsample_np = np.load("C:/Users/Zhiwen Yan/Desktop/23summer/input_features.npy")
```

Comment:
Can't have absolute paths like this one. It either need to be a parameter, or a relative path that will work every time.

Reply:
Contrastive part has not been generalized, just a test version on local machine.